content/en/docs/setup/production-environment/tools/kubeadm/install-kubeadm.md

```diff
@@ -78,7 +78,7 @@ For more details please see the [Network Plugin Requirements](/docs/concepts/ext
 
 | Protocol | Direction | Port Range | Purpose                 | Used By                   |
 |----------|-----------|------------|-------------------------|---------------------------|
-| TCP      | Inbound   | 6443*      | Kubernetes API server   | All                       |
+| TCP      | Inbound   | 6443\*      | Kubernetes API server   | All                       |
```

Comment:
is the \ before * required here (escaping)?

Reply:
Yes. My editor treats it as an opening for markdown emphasize `*something*` if not escaped.